internal-packages/next-dev/src/durableObjects.ts

```diff
@@ -9,18 +9,20 @@ import {
 } from './wrangler';
 import { warnAboutExternalBindingsNotFound } from './utils';
 
+export type DevBindingsDurableObjectOptions = Record<string, Extract<Binding, { type: 'durable-object' }>>;
+
 /**
  * Gets information regarding DurableObject bindings that can be passed to miniflare to access external (locally exposed in the local registry) Durable Object bindings.
  *
  * @param durableObjects
  * @returns the durableObjects and WorkersOptions objects to use or undefined if connecting to the registry and/or creating the options has failed
  */
 export async function getDOBindingInfo(
```

Comment:
this function's logic is a bit off, only the binding name is used to match the DO of external workers and not the scriptName nor the class name is used, not sure how I made such a blunder 😓, but it needs fixing

Reply:
amended